### ISSUE TITLE
When viewing current password, it does not show current number of characters

### ISSUE THREAD

**Issue by darkwing1** [keepassxreboot/keepassxc#3859]
[TIP]:  # ( Provide a general summary of the feature in the title above ^^ )
[TIP]:  # ( DO NOT include screenshots of your actual database! )

## Summary
[NOTE]: # ( Provide a brief overview of what the new feature is all about )
Currently i dont see anywhere where it would show the number of characters your password is set on an entry. If im not mistaken, this is possible on the keepass.info variant.

## Desired Behavior
[NOTE]: # ( Tell us how the new feature should work, be specific )
It would be nice to be able to see the current number of characters when viewing the password after opening up the entry, before setting a new password.

## Possible Solution
[NOTE]: # ( Not required, but suggest ideas on how to implement the addition or change )

## Context
[NOTE]: # ( Why does this feature matter to you? What unique circumstances do you have? )
Knowing the number of characters on your current password helps to determine by how many more character to set it when resetting/changing the password.
Currently i have to paste the current password in Wordpad for example to count the number of characters and set my new password accordingly.

Also i see that when pasting that current password in the generator, the generator does not automatically set the number of characters you have pasted.  Maybe this can be fixed as well?

Thank you for your great work.

v2.5.1

**Comment by droidmonkey** [comment]
The password generator sets the size of the generated password to the same length as the current one.

**Comment by darkwing1** [comment]
@droidmonkey no i dont see that happening here. on v2.5.1 currently this is what happens:

- I open KeepassXC
- I open an entry & unhide the password.
- I open the generator and unhide the password.
- The password from the generator is 5 characters and the password of the entry is 10 characters.
So the password generator did not set the size automatically when opening it.

**Comment by droidmonkey** [comment]
I forgot to mention I like your feature proposal. I would also throw in a visual indication of the current password quality similar to what the password generator does now.

**Comment by darkwing1** [comment]
Yes a visual indicator sounds great! Actually ive been wanting to mention that for a long time now.

Can you try reproduce this issue?  All you need to do is copy a password of different length to the generator and see that it wont change the count automatically.

**Comment by viharm** [comment]
Yes, some websites/services have an upper limit on the number of characters in a password. I like to create passphrases myself, so don't always rely on the built-in password generator. Such a numeric indicator will be very helpful in creating such passphrases.

**Comment by TeutonJon78** [comment]
It might be nice to also include the entropy value of the current password like KeePass does in it's "Quality" display.
